Gems/LyShine/Code/Source/UiCanvasComponent.h

```diff
@@ -127,8 +127,8 @@ class UiCanvasComponent
 
     bool GetIsRenderToTexture() override;
     void SetIsRenderToTexture(bool isRenderToTexture) override;
-    AZStd::string GetRenderTargetName() override;
-    void SetRenderTargetName(const AZStd::string& name) override;
+    AZ::Data::Asset<AZ::RPI::AttachmentImageAsset> GetAttachmentImageAsset() override;
```

Comment:
Can this return a const ref instead? Any particular reason why we need to return a copy of the asset?